resources/dicts/events/misc/general.json

```diff
@@ -1,3722 +1,3943 @@
         "m_c": {
-            "age": ["kitten"]
+            "age": ["adolescent", "young adult", "adult", "senior adult", "senior"],
+            "status": ["medicine cat"],
+            "skill": ["OMEN,0"]
+        },
+        "r_c": {
+            "trait": ["bloodthirsty"]
         },
-          "r_c": {
-            "age":["kitten"]
-          },
         "relationships": [
-          {
-            "cats_to": ["m_c"],
-            "cats_from": ["r_c"],
-            "mutual": true,
-            "values": ["platonic", "comfort", "trust"],
-            "amount": 10
-          },
-          {
-            "cats_to": ["m_c"],
-            "cats_from": ["r_c"],
-            "mutual": true,
-            "values": ["dislike"],
-            "amount": -10
-          }
+            {
+                "cats_to": ["m_c"],
+                "cats_from": ["r_c"],
+                "mutual": true,
+                "values": ["dislike"],
+                "amount": 20
+            }
         ]
     },
     {
-        "event_id": "gen_misc_any_accessorykitkit2",
+        "event_id": "gen_misc_war_medcat_woundedenemy_pos",
         "location": ["any"],
         "season": ["any"],
-        "sub_type": ["accessory"],
+        "sub_type": ["war"],
         "weight": 20,
-        "event_text": "m_c starts bringing a pretty acc_singular everywhere {PRONOUN/m_c/subject} {VERB/m_c/go/goes}, and r_c loudly complains that {PRONOUN/r_c/subject} {VERB/r_c/want/wants} one, too.",
-        "new_accessory": ["WILD", "PLANT"],
+        "event_text": "Whilst out collecting herbs, m_c stumbles upon a o_c_n warrior attempting to pack a wound with leaves. Their movements are slow and uncoordinated - without help, they'll soon be hunting with their ancestors. It would be so easy to just... walk past. But {PRONOUN/m_c/subject} can't. It's just a few pieces of cobweb from the branches and a few moments of {PRONOUN/m_c/poss} time... nobody has to know.",
         "m_c": {
-            "age": ["kitten"]
+            "status": ["medicine cat", "medicine cat apprentice"]
         },
-        "r_c": {
-          "age":["kitten"],
-          "trait": ["bullying", "impulsive", "attention-seeker"]
+        "other_clan": {
+            "changed": 1
+        }
+    },
+    {
+        "event_id": "gen_misc_war_medcat_woundedenemy_neg",
+        "location": ["any"],
+        "season": ["any"],
+        "sub_type": ["war"],
+        "weight": 20,
+        "event_text": "Whilst out collecting herbs, m_c stumbles upon a o_c_n warrior attempting to pack a wound with leaves. Their movements are slow and uncoordinated - without help, they'll soon be hunting with their ancestors. It would be so easy to just... walk past. But {PRONOUN/m_c/subject} can't. A few berries placed into their jaws with some soothing words... and that's one less o_c_n cat to deal with.",
+        "m_c": {
+            "status": ["medicine cat", "medicine cat apprentice"]
```

Comment:
i think some trait constraints would be good both here and in the positive version of this event.  i.e. bloodthirsty, vengeful, righteous could kinda go both ways, sneaky, ect.